pkg/scheduler/plugins/capacity/capacity.go

```diff
@@ -272,8 +272,8 @@ func (cp *capacityPlugin) OnSessionOpen(ssn *framework.Session) {
 	ssn.AddAllocatableFn(cp.Name(), func(queue *api.QueueInfo, candidate *api.TaskInfo) bool {
 		attr := cp.queueOpts[queue.UID]
 
-		free, _ := attr.realCapability.Diff(attr.allocated, api.Zero)
-		allocatable := candidate.Resreq.LessEqual(free, api.Zero)
+		futureUsed := attr.allocated.Clone().Add(candidate.Resreq)
```

Comment:
We should also add new case to cover this.

Reply:
Has existed in `TestEnqueueAndAllocatable`